lib/src/Pos/Node/API.hs

```diff
@@ -547,47 +553,209 @@ instance ToSchema (V1 Version) where
         pure $ NamedSchema (Just "V1Version") $ mempty
             & type_ .~ SwaggerString
 
+
+newtype SecurityParameter = SecurityParameter Int
+    deriving (Show, Eq, Generic, ToJSON, FromJSON, Arbitrary)
+
+instance Buildable SecurityParameter where
+    build (SecurityParameter i) = bprint shown i
+
+instance ToSchema SecurityParameter where
+    declareNamedSchema _ =
+        pure $ NamedSchema (Just "SecurityParameter") $ mempty
+            & type_ .~ SwaggerNumber
+            & maximum_ .~ Just (fromIntegral (maxBound :: Int))
+            & minimum_ .~ Just (fromIntegral (minBound :: Int))
```

Comment:
If the schema is ismorphic to the schema for numbers, you could just do:

```hs
declaredNamedSchema _ = 
    s <- declareNamedSchema (Proxy @Int)
    return $ s & name .~ (Just "SecurityParameter")
```

```hs
declareNamedSchema :: proxy a -> Declare (Definitions Schema) NamedSchema
```

`Declare` here is actually a `Monad` with an internal state, with all the already declared schemas.